main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -55,6 +55,33 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
     );
   };
 
+  var doAddColumnWithUrlOfMatchedEntities = function AddColumnWithUrlOfMatchedEntities (columnName,dismissDialog) {
+    promptForColumn(AddColumnWithUrlOfMatchedEntities);
+    var serviceUrl = null;
+    var service = null;
+    if (column.reconConfig) {
+        serviceUrl = column.reconConfig.service;
+    }
+    if (serviceUrl) {
+        service = ReconciliationManager.getServiceFromUrl(serviceUrl);
+    } 
+    if (service && service.view){
+     Refine.postCoreProcess
```

Comment:
This code should only be executed once the user has supplied the column name, which means after the success callback of the `promptForColumn` function is called.

Reply:
I have tried to resolve these in my new commits, also I am sorry for the mess due to forced pushes.